styles/mixins/_layout.scss

```
@@ -82,3 +83,27 @@ $-space: var.$layout-space;
     margin-left: $size !important;
   }
 }
+
+/**
+ * Position an element vertically and horizontally within the viewport
+ */
+@mixin fixed-centered {
+  z-index: 20;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+/**
+ * Semi-opaque overlay, full-viewport
+ */
+@mixin overlay {
+  z-index: 10;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

I have seen also implemented this way:
```suggestion
  bottom: 0;
  right: 0;
```

---

Sure, this seems fine! I'll update.